src/app/hooks/useViewTracker/index.jsx

```diff
@@ -31,12 +28,27 @@ const useViewTracker = (props = {}) => {
     statsDestination,
   } = useContext(EventTrackingContext);
   const { service } = useContext(ServiceContext);
+  const initObserver = async () => {
+    if (typeof window.IntersectionObserver === 'undefined') {
+      // Polyfill IntersectionObserver, e.g. for IE11
+      await import('intersection-observer');
+    }
+    const callback = changes => {
+      const someElementsAreInView = changes.some(prop('isIntersecting'));
```

Comment:
Just looking at this again, why is this checking that some of the entries returned from the api have a truthy `isIntersecting` prop, wouldn't be expect all entries to be intersecting? Also wouldn't we always expect a single entry as we are observing a single element?

Reply:
if you think of multiple items in a list that are being tracked by one observer instance and 2 of them are in view but then you scroll one of the item out of view then then `setIsInView` will be called and set the `isInView` to false even though one item is still in view.

The logic here is that at least one tracked item should be in view to trigger the view event.

Reply:
> Also wouldn't we always expect a single entry as we are observing a single element?

In example 2 from the [useViewTracker README](https://github.com/bbc/simorgh/blob/latest/src/app/hooks/useViewTracker/README.md) we can track multiple items with one instance.